README.md

```diff
@@ -133,6 +133,8 @@ indexing, so the value is `1`, `2`, `3`, etc.
 
 - Android supports a maximum of three buttons, and ignores any more than that.
 
+- Android dialog title cannot exceed 2 lines of content, it will ignore any more than this.
```

Comment:
Was this always the case on Android or is this maybe a recent change?

Reply:
It was always the case. I have never seen it more than two lines in any Android platform or version that I worked on. Other people talk about this limitation in StackOverflow as well:
https://stackoverflow.com/questions/9107054/how-to-build-alert-dialog-with-a-multi-line-title/15847538

Reply:
can we merge it to close issue #108? Thanks